platforms/ide/problems/build.gradle.kts

```diff
@@ -23,11 +23,40 @@ description = """Problem SPI implementations.
     |This project contains the SPI implementations for the problem reporting infrastructure.
 """.trimMargin()
 
+
+val problemReportReportPath by configurations.creating {
+    isVisible = false
+    isCanBeConsumed = false
+    attributes { attribute(DocsType.DOCS_TYPE_ATTRIBUTE, objects.named("configuration-cache-report")) }
+}
+
+// You can have a faster feedback loop by running `configuration-cache-report` as an included build
+// See https://github.com/gradle/configuration-cache-report#development-with-gradlegradle-and-composite-build
+dependencies {
+    problemReportReportPath(libs.configurationCacheReport)
+}
+
+tasks.processResources {
+    from(zipTree(problemReportReportPath.elements.map { it.first().asFile })) {
+        into("org/gradle/internal/impl/problems")
+        exclude("META-INF/**")
+        rename { fileName ->
+            fileName.replace("configuration-cache-report", "problems-report")
```

Comment:
🤔 Why have a copy of the original template?

Reply:
For now, one of the next steps will be adapting the template